applications/crossbar/src/modules/cb_rates.erl

```diff
@@ -133,6 +150,23 @@ put(Context) ->
 delete(Context, _RateId) ->
     crossbar_doc:delete(Context).
 
+-spec validate_number(ne_binary(), cb_context:context()) -> cb_context:context().
+validate_number(Phonenumber, Context) ->
+    case length(re:replace(Phonenumber, "[^0-9]", "", [global, {return, list}])) >= 7 of 
```

Comment:
@onnet The whole point of the reconcile_regex is to distinguish between globally-routable and locally-routable numbers. I'm all for having a better, all-inclusive default, but that doesn't relieve the kazoo administrator of the responsibility of making sure they have a proper reconcile_regex configued.

The first thing hotornot does is run `wnm_util:to_e164/1` on the numbers supplied. So this work is being done on the query anyway; I don't see the problem of restricting the API in a similar way as callflows would when routing a number.

Anyway, we're happy to have a pull request with better defaults too! I think the call to `is_reconcilable/1` is more in line with how a kazoo cluster is expected to be configured for numbers than just checking length after a regex. Why waste time rating a number if the system doesn't believe it is reconcilable anyway?